DEVELOPER.md

```diff
@@ -55,15 +55,28 @@ with the `envoyproxy/gateway-dev:latest` Envoy Gateway image into a Kubernetes c
 **_NOTE:_**  Replace `IMAGE` with your registry's image name.
 
 ### Run Gateway API Conformance Tests
-* Run `make conformance` to run Gateway API Conformance tests using `envoy-gateway` in a
-local Kind cluster. Go [here](https://gateway-api.sigs.k8s.io/concepts/conformance/) to learn
-more about the tests.
+The commands below build and push the Envoy Gateway image to your hub, deploy Envoy Gateway to the Kubernetes cluster,
+and run the Gateway API conformance tests. Refer to the Gateway API [conformance homepage][conform] to learn more about
+the tests.
 
-**_NOTE:_** Conformance tests against a kind cluster is currently unsupported on Mac computers.
-As a workaround, you could run this against your own Kubernetes cluster (such as Kubernetes on Docker Desktop) using this command -
-`IMAGE=docker.io/you/gateway-dev make push-multiarch && IMAGE=docker.io/you/gateway-dev make kube-deploy && make run-conformance`
-which builds and pushes the Envoy-Gateway image to your hub, deploys Envoy Gateway resources into your cluster
-and runs the Gateway API conformance tests.
+#### On a Linux Host
+* Run `make conformance` to run Gateway API conformance tests against Envoy Gateway in a local Kind cluster.
+
+#### On a Mac Host
+Since Mac doesn't support [directly exposing][kind_lb] the Docker network to the host, use one of the following
+workarounds to run conformance tests:
+
+- Run Envoy Gateway in your own Kubernetes cluster and then run `IMAGE=docker.io/you/gateway-dev make push-multiarch &&
+IMAGE=docker.io/you/gateway-dev make kube-deploy && make run-conformance`
+
+- Run Docker Desktop with [Kubernetes support][docker_kube] and then run `IMAGE=docker.io/you/gateway-dev make
+push-multiarch && IMAGE=docker.io/you/gateway-dev make kube-install-image && IMAGE=docker.io/you/gateway-dev make
+kube-deploy && make run-conformance`
```

Comment:
Can combine make commands:

```suggestion
- Run Docker Desktop with [Kubernetes support][docker_kube] and then run `IMAGE=docker.io/you/gateway-dev make
push-multiarch kube-install-image kube-deploy run-conformance`
```

Reply:
Commit 434bea6 greatly simplifies this section and Make targets, PTAL.